README.md

```diff
@@ -152,27 +181,32 @@ For example, to run the tests for the project `api`, run
 
 To run a single test, use the `testFile` flag as follows:
 
-> > nx run <project-name>:test --testFile <name-of-test-file>
+> > nx text <project-name> --test-file=<name-of-test-file>
```

Comment:
nx test

Reply:
Good catch- I'll try all commands and double check them before I update the readme.